src/pages/BaseItems.page.js

```diff
@@ -0,0 +1,42 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
+
+export class BaseItemsPage extends BaseSwagLabPage {
+
+    itemSelector = '.cart_item';
+
+    itemNameSelector = '.inventory_item_name';
```

Comment:
I dont see any reason to have this as a separate property since you use it just once.